examples/encryption_bot.ts

```diff
@@ -34,7 +36,7 @@ const worksImage = fs.readFileSync("./examples/static/it-works.png");
 const client = new MatrixClient(homeserverUrl, accessToken, storage, crypto);
 
 (async function() {
-    let encryptedRoomId: string;
+    let encryptedRoomId: string|undefined = undefined;
```

Comment:
under the current TS guidelines for the project, the line was already correct. The explicit undefined-ness is noisy rather than helpful, imo.

Reply:
For some reason, this spooks vscode but I trust your judgement.